### PR TITLE
Fix134

### PR DESCRIPTION
**Pull request by Aniket21mathur**

### CODE REVIEW

**Comment by afeena** [comment]
@Aniket21mathur  could you please resolve conflicts first?

File: bin/clone [hunk 1]
```diff
@@ -38,15 +38,22 @@ def main():
...
+    parser.add_argument("--path", help="path to save the page to be cloned", required=False,default="/opt/")
     args = parser.parse_args()
+    default_path = os.path.join(args.path, 'snare')
     if args.log_path:
         log_err = args.log_path + "clone.err"
     else:
-        log_err = "/opt/snare/clone.err"
+        log_err = os.path.join(default_path, 'clone.err')
+        
+    if not os.path.isfile(log_err):
```
Comment (afeena) [inline bin/clone]: you can check dir for `default_path` instead

File: snare/tests/test_add_meta_tag.py [hunk 2]
```diff
@@ -21,7 +21,8 @@ def setUp(self):
     def test_add_meta_tag(self):
         config = configparser.ConfigParser()
         config['WEB-TOOLS'] = dict(google='test google content', bing='test bing content')
-        add_meta_tag(self.page_dir, self.index_page, config)
+        default_path = '/opt/snare'
```
Comment (afeena) [inline snare/tests/test_add_meta_tag.py]: for test I'd better use smth as tmp

**Comment by afeena** [comment]
@Aniket21mathur please resolve conflicts

File: snare/tests/test_add_meta_tag.py (hunk 2 above)
Comment (puneet29) [reply snare/tests/test_add_meta_tag.py]: @afeena can you elaborate on what are you trying to suggest? I didn't get you.

**Comment by puneet29** [comment]
Hey I'd like to work on this if no one is working? Seems like @Aniket21mathur is not responding.

**Comment by Aniket21mathur** [comment]
> Hey I'd like to work on this if no one is working? Seems like @Aniket21mathur is not responding.

Hey, I was a bit involved in other work in the past few days. Now I am free to make changes. I notice you have opened a new pull request, so I will request @afeena to decide which one to work upon :smile: .

Thanks!

**Comment by puneet29** [comment]
Let's see what @afeena says. I'm sorry though @Aniket21mathur 😅

**Comment by afeena** [comment]
@Aniket21mathur  Since you've started to work on that issue earlier and I was busy and didn't check it before, I would prefer you to continue on that pull request :)

@puneet29 we have a similar problem in tanner, would you mind to work on this problem? https://github.com/mushorg/tanner/pull/305

**Comment by puneet29** [comment]
Sure I will work on that issue, @afeena .

**Comment by afeena** [comment]
@puneet29 Awesome!

**Comment by afeena** [comment]
@puneet29 Awesome!

**Comment by Aniket21mathur** [comment]
@puneet29 please proceed with this issue as well :smile:. Thanks!

**Pull request closed without merging**